client/src/includes/messages.test.js

```diff
@@ -0,0 +1,99 @@
+import { initButtonSelects } from './initButtonSelects';
+
+// save our DOM elements to a variable
+const testElements = `
+<div class = "button-select">
+  <input type="hidden" value/>
+  <button class="button-select__option" value>
+    All
+  </button>
+  <button class="button-select__option" value = "true">
+    Awaiting my review
+  </button>
+</div>
+
+<div class="button-select">
+  <input type="hidden" value />
+  <button class="button-select__option" value>
+    All
+  </button>
+  <button class="button-select__option" value="in_progress">
+    In Progress
+  </button>
+  <button class="button-select__option" value="approved">
+    Approved
+  </button>
+  <button class="button-select__option" value="needs_changes">
+    Needs changes
+  </button>
+  <button class="button-select__option" value="cancelled">
+    Cancelled
+  </button>
+</div>
```

Comment:
I would skip this whole additional select with multiple options, no need for now, just a really simple test is fine.